PrintManual.cpp

```diff
@@ -0,0 +1,55 @@
+#include "PrintManual.h"
+#include <iostream>
+
+const int COLORCELLSPACES = 11;
+const int ONEDIGITSPACES = 11;
+const int TWODIGITSPACES = 10;
+const int MAX_COLOR_COMBINATIONS = TelCoColorCoder::numberOfMajorColors * TelCoColorCoder::numberOfMinorColors;
+std::string SEPARATOR = "==========================================";
+
+void getColorNameSizes(int& majorColorSize, int& minorColorSize, TelCoColorCoder::ColorPair& colorPair) {
+    std::string colorStr;
+
+    colorStr = TelCoColorCoder::MajorColorNames[static_cast<int>(colorPair.getMajor())];
+    majorColorSize = colorStr.size();
+    colorStr = TelCoColorCoder::MinorColorNames[static_cast<int>(colorPair.getMinor())];
+    minorColorSize = colorStr.size();
+}
+void fillSpaces(int numberofSpaces)
```

Comment:
Unlike the above function, `fillSpaces` is not easily testable: We need to check the console output. If we separate the forming of the string from the console-output, does it become easier to test?